app/Http/Controllers/DeclarationsController.php

```diff
@@ -167,40 +167,43 @@ public function bulkStore(BulkDeclarationsRequest $request)
 				$request->file("data.${key}.image")
 			);
 			$this->applyFileData($data, $fileData);
-			
+
 			Declaration::create($data);
 		}
-		
+
 		$request->session()->flash(
 			'flash_message', 'De declaraties zijn opgeslagen!'
 		);
 
 		return response()->json(["status" => "success"]);
 	}
-	
-	
+
+
 	public function admin()
 	{
 		$this->authorize('viewAll', Declaration::class);
-		$members = Member::whereHas("declarations", function($q) {
-			$q->whereNull('closed_at');
-		})->get();
-		$total_open = Declaration::open()->where('gift',0)->sum('amount');
-		
-		return view('declarations.admin', compact('members', 'total_open'));
+        $openDeclarations = Declaration::query()
```

Comment:
Indenting seems off

Reply:
Tabs vs spaces you ganna hate me for doing what I am about todo